cookbooks/ros2_windows/recipes/openssl.rb

```diff
@@ -1,17 +1,32 @@
+openssl_versions = {
+  "dashing" => "1_0_2u",
+  "eloquent" => "1_0_2u",
+  "foxy" => "1_1_1g",
+  "rolling" => "1_1_1g",
+}.freeze
+
+openssl_version = openssl_versions[node["ros2_windows"]["ros_distro"]]
+openssl_conf_dir = if openssl_version == "1_1_1g"
+                     'C:\Program Files\\OpenSSL-Win64'
```

Comment:
Your use of backslashes here is inconsistent. `\` or `\\`? 

Also, is the expected case for all versions >=1.1.1g? Maybe for simpler maintenance, the `if` should check if it's 1.0.2u otherwise it defaults to the Program Files path?

Reply:
Yeah, to be honest I have no idea what the criteria are since we don't own the binaries and the pipeline isn't public. I like the idea of special casing the earlier release though. 

One thing I should add a comment for is that if an earlier openssl install was present, even if it has been uninstalled, the default installation directory seems to revert. I haven't found a good way to detect this but a significant enhancement would be to force the use of one directory or the other for all SSL releases.